indico/modules/auth/blueprint.py

```diff
@@ -25,7 +28,6 @@
 
 _bp.add_url_rule('/login/', 'login', multipass.process_login, methods=('GET', 'POST'))
 _bp.add_url_rule('/login/<provider>', 'login', multipass.process_login, methods=('GET', 'POST'))
-
 _bp.add_url_rule('/logout/', 'logout', RHLogout, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/user/identities/associate', 'associate_identity', RHAssociateIdentity, methods=('GET', 'POST'))
```

Comment:
ass-ciate? :D

I'm fine with link/unlink, too. No strong opinion for either wording here.